course_discovery/apps/course_metadata/data_loaders/api.py

```diff
@@ -1016,6 +1016,15 @@ def _add_course_subjects(self, categories, course_run):
                 subject.slug = category['slug']
                 subject.save()
 
+            if subject:
+                for language_code, translated_title in category['title_translations'].items():
+                    subject_translation, __ = SubjectTranslation.objects.get_or_create(
+                        master_id=subject.pk,
+                        language_code=language_code
+                    )
+                    subject_translation.name = translated_title if translated_title else category['title']
```

Comment:
also, we should check if `title` is present or use `get` method

Reply:
@taimoor-ahmed-1 translated_title is optional while a category cannot be created without title hence we don't need to check if title is present or not.